client/my-sites/sidebar/sidebar.jsx

```diff
@@ -308,14 +308,20 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		const planLink = '/plans' + this.siteSuffix(),
-			planName = ( this.isSingle() ) ? site.plan.product_name_short : '';
+		const planLink = '/plans' + this.siteSuffix();
 
 		let linkClass = 'upgrades-nudge';
+
 		if ( productsValues.isPlan( site.plan ) ) {
 			linkClass += ' is-paid-plan';
 		}
 
+		let planName = site.plan.product_name_short;
```

Comment:
Did you ever work out why there was a call to `this.isSingle()` previously? It seems to not be needed, but I wonder if there's an explanation...